src/lib/components/NewLayeredMap/layers/L.imageWebGLOverlay.js

```diff
@@ -112,8 +112,9 @@ L.ImageWebGLOverlay = L.Layer.extend({
 	    
     	if(options.colorScale && options.colorScale instanceof Object && this.options.colorScale instanceof Object) {
     		options.colorScale = Object.assign(this.options.colorScale, options.colorScale)
-   	}       	   
+   	    }       	   
```

Comment:
Not exactly an error, but maybe line up the  } when checking other stuff?

Reply:
I will fix these in interns-dev